test/e2e/page-objects/pages/confirmations/redesign/gas-fee-token-modal.ts

```diff
@@ -0,0 +1,51 @@
+import { Driver } from '../../../../webdriver/driver';
+
+export default class GasFeeTokenModal {
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  protected driver: Driver;
+
+  private readonly listItem = (symbol: string) =>
+    `[data-testid="gas-fee-token-list-item-${symbol}"]`;
+
+  private readonly listItemSymbol =
```

Comment:
small nit: could this be ordered alphabetically?

Reply:
Done.